content/news/025/index.md

```diff
@@ -599,8 +592,8 @@ aims to be easy to use and provide a large set of out-of-the-box features.
 [![rg3d feature highlights](rg3d-feature-highlights.png)][rg3d_fh]
 __Engine feature highlights__
 
-Check out the video about the engine made by [@GameFromScratch][gfs_channel],
-an article could be found [here][gfs_article].
+Check out the video about the engine made by [@GameFromScratch][gfs_channel], an
```

Comment:
There're three images in rg3d's section - the logo and two previews for videos - while the guideline is one image per section.

I propose removing the logo and replacing the GFS's video with just a text link.

Also, the list is formatted as a source code - this gors against the style guide as it'll cause horizontal scroll on mobile.